packages/editor/package.json

```diff
@@ -37,7 +37,7 @@
     "watch:client": "webpack-dev-server --mode development --hot --inline",
     "watch:server": "run-p watch:server:*",
     "watch:server:start": "nodemon -r dotenv/config --inspect=0.0.0.0:9230 ./dist/server/index.js",
-    "watch:server:build": "tsc -b ./src/shared ./src/server --watch",
+    "watch:server:build": "tsc -b ./src/shared ./src/server ./src/client --watch",
```

Comment:
If you want to have this add another command for consistency.
`"watch:client:build": "tsc -b ./src/client --watch"` plus `watch:client:start` and rework `watch:client`...